File-Formats/PDF/Convert-HTML-To-PDF/features.md

```diff
@@ -1505,7 +1261,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## HTML to single PDF page
 
-By using this option, you can render the whole HTML content into a single PDF page. The PDF page size is limited to 14400 points. There are two options to enable this feature since this is disabled by default.
+By using this [SinglePageLayout](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.HtmlToPdf.SinglePageLayout.html) Enum, you can render the whole HTML content into a single PDF page. The PDF page size is limited to 14400 points. There are two options to enable this feature since this is disabled by default.
```

Comment:
use the SinglePageLayout property here, not enum

Reply:
modified